doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib

```diff
@@ -354,6 +354,35 @@ The list of builtin attributes is as follows:
 Like regular attributes, they can be  used in `q` filters and in `orderBy`.
 However, they cannot be used in resource URLs.
 
+## Special Metadata Types
+
+Similar to attributes, each piece of metadata has a metadata name, a metadata type, a metadata value.
+The metadata field can also be used to explicitly include some special metadata e.g. dateCreated,
+dateModified and dateExpires i.e. for attribute creation date, last modification date and expiration date
+respectively (These are not included by default).
+
+* `DateTime`:  identifies dates, in ISO8601 format. This Metadata can be used with the query
+  operators greater-than, less-than, greater-or-equal, less-or-equal and range. For instance
+  (only the referred attribute metadata is shown):
+
+```
+"metadata": {
+      "dateCreated": {
+        "value": "2019-09-23T03:12:47.213Z",
+        "type": "DateTime"
+      }
+}
+```
+
+Datetimes are composed of date, time and timezone designator, in one of the following patterns:
+<date>
+<date>T<time>
+<date>T<time><timezone>
+
+Orion always provides datetime metadata using the format YYYY-MM-DDThh:mm:ss.ssZ. It uses UTC/Zulu timezone 
+(which is the best default option, as clients/receivers may be running in any timezone). 
+For more details about DateTime follow [DateTime section](https://fiware-orion.readthedocs.io/en/master/user/ngsiv2_implementation_notes/index.html)
```

Comment:
By similarity with "Special Attribute Types" section, I'd suggest to delete this block of lines.

Reply:
Fixed in 0a39e5b in PR #3654